stdlib/Profile/src/Profile.jl

```diff
@@ -5,7 +5,11 @@ Profiling support, main entry point is the [`@profile`](@ref) macro.
 """
 module Profile
 
-import Base.StackTraces: lookup, UNKNOWN, show_spec_linfo, StackFrame
+import Base.StackTraces: lookupat, UNKNOWN, show_spec_linfo, StackFrame
+
+# deprecated functions: use `getdict` instead
+lookup(ip::UInt) = lookupat(convert(Ptr{Cvoid}, ip) - 1)
```

Comment:
Yeah, it's to make it easier for consumers to detect the change, if necessary. And the deprecation here is because it's where ProfileView expected it. OTOH, this would now subtract 2 for ProfileView (since fetch is now already returning "corrected" values), so perhaps I should define `const lookup = lookupat`? or ProfileView should switch to using `getdict` sometime before the v1.4 release?